susePublicCloudInfoClient/pint

```diff
@@ -56,8 +56,8 @@ import sys
 
 from docopt import docopt
 
-import infoserverrequests
-import version
+import lib.susepubliccloudinfoclient.infoserverrequests as ifsrequest
+import lib.susepubliccloudinfoclient.version as version
 
```

Comment:
This is not going to work once installed. Once installed into sitelib the lib directory disappears, it is just an artifact for the code organization in development. Installed on a system all code will live only under the susepubliccloudinfoclient directory, which becomes the package in Python